spec/translations/translationDelete.js

```diff
@@ -0,0 +1,106 @@
+const { describe, it, before, beforeEach, afterEach } = require("mocha");
+const mongoose = require("mongoose");
+const request = require("supertest");
+const { expect } = require("chai");
+
+const createHttpError = require("../../utils/createHttpError");
+const { RESULT } = require("../../constants/responseMessages");
+const { TRANSLATIONS } = require("../../constants/error");
+const Translation = require("../../models/Translation");
+const app = require("../../app");
+
+describe.only("delete translation test", function cb() {
+  this.timeout(10000);
+
+  const db = mongoose.connection;
+  let translationId = null;
+
+  const createTranslation = (done) => {
+    const translationData = {
+      user: "test@gmail.com",
+      origin:
+        "As your app grows, you can catch a lot of bugs with typechecking.",
+      translated:
+        "앱이 성장함에 따라 유형 검사로 많은 버그를 잡을 수 있습니다.",
+      url: "https://www.naver.com",
+      glossary: {
+        react: "리액트",
+      },
+    };
+
+    (async () => {
+      try {
+        const translation = await Translation(translationData).save();
+
+        translationId = translation.id;
+
+        return done();
+      } catch (error) {
+        return done(error);
+      }
+    })();
+  };
+
+  const deleteTranslation = (done) => {
+    (async () => {
+      try {
+        await Translation.findOneAndDelete({ user: "test@gmail.com" });
```

Comment:
```suggestion
        await Translation.findByIdAndDelete(translationId);
```
이렇게 하면 좀 더 의존성 없는 테스트 코드가 될 것 같습니다

Reply:
`translationId`를 다른곳에서 필요해서 쓰기 전에 작성 해 뒀던 것인데 누락되었네요.

확인 감사합니다 !